app/modules/site_settings/models.py

```diff
@@ -428,19 +398,12 @@ def get_boolean(cls, key, default=None):
             return cls._get_default_value(key)
         return setting.boolean if setting else default
 
-    # a bit of hackery.  right now *all* keys in edm-configuration are of the form `site.foo` so we use
-    #   as a way branch on _where_ to get the value to return here.  but as we ween ourselves off edm config,
-    #   either gradually or forklift, this can be extended to allow that
-    @classmethod
-    def is_edm_key(cls, key):
-        return key.startswith(cls.EDM_PREFIX)
-
     @classmethod
     def get_value(cls, key, default=None, **kwargs):
```

Comment:
Again you have a valid point and yes, this might get revisited 'real soon' but I'd like to get this in as is.

Reply:
i was considering this exact thing when i added the `get_boolean` ... yes lets revisit.  i probably ran into the same small edge cases that made me balk on it.  but i think it would be desirable, especially when we know we can trust the _type_ property to hint us on what to actually get.